kuma/static/styles/locales/ja.scss

```diff
@@ -46,10 +46,8 @@ Styles for Japanese
 
 /* Bug 973171 */
 /* stylelint-disable declaration-no-important */
-* {
-    /* !important required for locale specific override */
-    font-style: normal !important;
-}
+/** Back ground of Bug 973171 is obsolete already, the fix was removed. 
+    It was for Windows XP, lacks Anti-Alias function **/
```

Comment:
@hmatrjp Thank you for the update. I believe you can safely remove this comment as well. The bug and this pull request are there to keep the history. I will test this locally and let you know if I see any other problems. Thank you!

Reply:
Hi,
did you finish the test locally? @schalkneethling 
Or should I remove the comment first?

Reply:
You can safely remove the comment. Thx @hmatrjp

Reply:
I removed comment. Thanks. @schalkneethling